### PR TITLE
OCPQE-11259: Add build info to artifacts

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by liangxia** [comment]
We can get below version info in `artifacts/<job-name>/cucushift-helper-service-build-info/artifacts/ocp-build-info`
```
4.12.0-0.nightly-2022-07-26-131732
```
It matches the version we got from the build log,
```
INFO[2022-07-27T02:50:34Z] ci-operator version v20220727-a5ba2e724      
INFO[2022-07-27T02:50:34Z] Resolved source https://github.com/openshift/release to master@2857a953, merging: #30814 68a76060 @liangxia 
...
INFO[2022-07-27T02:50:34Z] Resolved release latest to registry.ci.openshift.org/ocp/release:4.12.0-0.nightly-2022-07-26-131732
```

File: ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -n "${RELEASE_IMAGE_LATEST}" ]]; then
+    echo "Release info for: ${RELEASE_IMAGE_LATEST}"
+    oc adm release info "${RELEASE_IMAGE_LATEST}" || true
+    PAYLOAD_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
+    echo "${PAYLOAD_VERSION}" > "${ARTIFACT_DIR}/ocp-build-info"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh:11]: Looks good, is it necessary that we also add such info for upgrade jobs

Comment (liangxia) [reply ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh:11]: upgrade needs extra effort than e2e, will do that in another PR.

**Review by dis016 — COMMENTED** [review]
/lgtm

File: ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh (hunk 1 above)
Comment (liangxia) [reply ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh:11]: Add the info for upgrade in https://github.com/openshift/release/pull/30881

[the remark this answers is not in the document]
Comment (liangxia) [reply ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh:11]: I've tried to add `RELEASE_IMAGE_TARGET` with `[[ -n "${RELEASE_IMAGE_TARGET}" ]]`, and it show me an error `/bin/bash: line 17: RELEASE_IMAGE_TARGET: unbound variable`, so I got it removed in this PR.

Follow up for that in https://github.com/openshift/release/pull/30881

**Pull request merged**